Dockerfile

```
@@ -21,7 +24,19 @@ COPY --from=build-epicli-wheel /src/core/src/epicli/dist/ /epicli/
 
 RUN apt-get update \
     && apt-get install --no-install-recommends -y \
-        gcc libffi-dev make musl-dev openssh-client ruby-full sudo tar unzip vim \
+        curl gcc libffi-dev make musl-dev openssh-client ruby-full sudo tar unzip vim \
+\
+    && echo "Installing helm binary ..." \
+    && curl -LOsS https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz \
```

"-f" would be nice here, otherwise if there is some error and data cannot be downloaded curl will return 0 anyway and docker build will not fail.
From https://curl.haxx.se/docs/manpage.html we can read:
```
-f, --fail

(HTTP) Fail silently (no output at all) on server errors. This is mostly done to better enable scripts etc to better deal with failed attempts. In normal cases when an HTTP server fails to deliver a document, it returns an HTML document stating so (which often also describes why and more). This flag will prevent curl from outputting that and return error 22.

This method is not fail-safe and there are occasions where non-successful response codes will slip through, especially when authentication is involved (response codes 401 and 407).
```
```suggestion
    && curl -fLOsS https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz \
```

---

Good point, thanks! 👍